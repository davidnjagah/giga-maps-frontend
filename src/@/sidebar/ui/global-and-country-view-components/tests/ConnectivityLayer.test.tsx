import { render } from '@testing-library/react';
import { testWrapper } from '~/tests/jest-wrapper';
import ConnectivityLayer from '../connectivity-layer/connectivity-layer.view';
jest.mock('@carbon/charts-react', () => ({
  SimpleBarChart: jest.fn().mockReturnValue(null),
}))


describe('ConnectivityLayer', () => {

  it('should render component', () => {
    const { asFragment } = render(testWrapper(<ConnectivityLayer />));
    expect(asFragment).toMatchSnapshot();
  });

});
