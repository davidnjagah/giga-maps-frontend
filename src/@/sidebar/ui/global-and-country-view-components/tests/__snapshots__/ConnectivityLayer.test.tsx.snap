// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConnectivityLayer should render component 1`] = `[Function]`;
